examples/gallery/plot/datetime-inputs.py

```diff
@@ -0,0 +1,58 @@
+"""
+Datetime inputs
+---------------
+
+The :meth:`pygmt.Figure.plot` method can plot datetime inputs of individual types:
+
+- numpy.datetime64: for available options see https://numpy.org/doc/stable/reference/arrays.datetime.html
```

Comment:
Can we change it to 
```
:class:`numpy.datetime64`
```
?

I'm not sure if it works.

Reply:
> 
> 
> Can we change it to
> 
> ```
> :class:`numpy.datetime64`
> ```
> 
> ?
> 
> I'm not sure if it works.

I didn't find any class definition as for the pandas or xarray example, so I'm not sure if it would work. But we can give it a try.

Reply:
No it's not possible to directly link it, do you have a suggestion how to handle it @seisman ?

Reply:
Looks like an upstream issue https://github.com/numpy/numpy/issues/10039

Reply:
ok, means we leave it as is so far?